ftag/flavours.yaml

```diff
@@ -140,6 +140,43 @@
   colour: "#e76f51"
   category: lepton-decay
 
+# PGID labelling: taken from https://pdg.lbl.gov/2023/reviews/rpp2023-rev-monte-carlo-numbering.pdf
+- name: D0meson
+  label: $D^0$ in $c$-hadron decay
+  cuts: ["HadronConeExclTruthLabelID == 4", "HadronConeExclTruthLabelPdgId in (421,-421)"]
+  colour: "#FFA500"
+  category: PGID
```

Comment:
Can we use `PDGID` instead of `PGID` everywhere for consistency with the label name?